app/src/test/java/sw19/moring03/paint/SprayCanTest.java

```diff
@@ -0,0 +1,33 @@
+package sw19.moring03.paint;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.PointF;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import sw19.moring03.paint.tools.SprayCanTool;
+
+import static junit.framework.TestCase.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SprayCanTest {
```

Comment:
have a look at the tests of the other tools like pathTool

atleast an invalid test where draw returns false is needed. 
All your functions with all there paths should be tested. (Code Coverage should go up not down ;))